oidc-conformance-tests/send_chat.py

```diff
@@ -46,20 +47,72 @@
     if len(failed_tests_list['others']) > 0:
         total_tests_count += len(failed_tests_list['others'])
 
+font_color = '#009944'
+if workflow_status == 'failure':
+    font_color = '#ff0000'
+
+message_headers = {'Content-Type': 'application/json; charset=UTF-8'}
+message = {
+    "cards": [
+        {
+            "header": {
+                "title": "OIDC Conformance Test",
+                "subtitle": "GitHub Action #" + github_run_number
 
-# send google chat notification
-request_body = {
-    'text': 'Hi all, OIDC conformance test run #' + github_run_number + ' for IS ' + str(wso2_is_version) +
-            ' completed with status: ' + workflow_status +
-            ' \n Total test cases: ' + str(total_tests_count) +
-            ' \n Failed test cases: ' + str(failed_count) +
-            ' \n Test cases with warnings: ' + str(warnings_count) +
-            ' \n https://github.com/' + github_repository_name + '/actions/runs/' + github_run_id
+            },
+            "sections": [
+                {
+                    "widgets": [
+                        {
+                            "textParagraph": {
+                                "text": "<b>Identity Server " + str(wso2_is_version) + "</b>"
```

Comment:
Shall we use f-strings instead of concatenation here to improve the readability?

Reply:
addressed with a new PR